tests/integration/conftest.py

```diff
@@ -21,7 +57,24 @@ def copy_loki_library_into_tester_charm(ops_test):
     shutil.copyfile(library_path, install_path)
 
 
+@pytest.fixture(scope="module", autouse=True)
+def copy_sample_alerts_into_tester_charm(ops_test):
```

Comment:
How about a helper function to update the contents of a zip file as an alternative to this?

Reply:
I very much do not think that we should build test infrastructure around assumptions about the internal structure of charms (or any other archive we don't control).

This pattern is already used for libs in multiple charms, and if checksumming is ever added anywhere, it will work with that, too, because it's pre-build. What is the advantage of trying to update a zip?